.github/workflows/release-wasm-bindings-to-npm.yml

```diff
@@ -0,0 +1,42 @@
+name: Wasm publish to NPM
+
+on: workflow_dispatch
+
+jobs:
+  publish-wasm:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: bindings/wasm
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+          target: 'wasm32-unknown-unknown'
+
+      # Download a pre-compiled wasm-bindgen binary.
+      - name: Install wasm-bindgen-cli
+        uses: jetli/wasm-bindgen-action@24ba6f9fff570246106ac3f80f35185600c3f6c9
+        with:
+          version: '0.2.84'
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install Yarn
+        run: npm i -g yarn
+
+      - name: Install JS dependencies
+        run: yarn
+
+      - name: Build project
+        run: yarn build
```

Comment:
This doesn't look like it's releasing?

Reply:
I removed the release ending for testing the gh actions